data.json

```
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "FSFE",
+    "difficulty": "easy",
+    "description": "Fill a form and receive promotional materials from Free Software Foundation Europe!",
```

I'd rephrase it like:
```suggestion
    "description": "Fill the form to receive promotional materials from Free Software Foundation Europe!",
```
WDYT?